packages/notifications/package.json

```
@@ -24,6 +24,7 @@
   ],
   "dependencies": {
     "@terra-ui-packages/components": "^0.0.6",
+    "@terra-ui-packages/core-utils": "*",
```

```suggestion
    "@terra-ui-packages/core-utils": "^0.1.0",
```

For anything that's going to be published, let's use actual version specifiers instead of `*`. `*` is really only for the Terra UI application, which always uses the latest version in the repository.